pymodm/fields.py

```diff
@@ -766,7 +766,10 @@ def __init__(self, field=None, verbose_name=None, mongo_name=None,
         :parameters:
           - `verbose_name`: A human-readable name for the Field.
           - `mongo_name`: The name of this field when stored in MongoDB.
-          - `field`: The Field type of all items in this list.
+          - `field`: The Field type of all items in this list. 
+
+        .. note:: 'field' needs to be an 'Field instance' and not a 
```

Comment:
Instead of adding a separate note, I think we can include this information in the description of the `field` parameter. 
```
          - `field`: The Field type of all items in this list. This needs to be an *instance* of a `MongoBaseField` subclass. 
```